xprof/btx_timeline.cpp

```diff
@@ -482,17 +533,7 @@ void btx_finalize_component_callback(void *usr_data) {
     }
   }
 
-  std::string path{dispatch->params->output_path};
-  if (path.empty()) {
-    path = "out.pftrace";
-  }
-
-  // Write the new address book back to disk.
-  std::fstream output(path, std::ios::out | std::ios::trunc | std::ios::binary);
-  if (!dispatch->trace.SerializeToOstream(&output))
-    std::cerr << "THAPI: Failed to write the trace at location: " << path << std::endl;
-  else
-    std::cout << "THAPI: Perfetto trace location: " << path << std::endl;
```

Comment:
We lost the messaging about the "trace location".

Reply:
Indeed fixed in `7bfa9e6` !